ledger-tool/src/main.rs

```diff
@@ -2746,6 +2755,7 @@ fn main() {
                     );
                     exit(1);
                 }
+                process_options.halt_at_slot = Some(snapshot_slot);
```

Comment:
🤔 this needs to be here because we need the process_options to get access_type for blockstore, and need the blockstore for figuring out the snapshot slot.

I don't see a better way, just commenting.